src/client/getConnectPublicViemClient.ts

```diff
@@ -52,21 +51,19 @@ export type ConnectClient<
 
 export type ConnectClientParams = {
   wallet: ComethWallet
-  apiKey: string
   rpc?: string
 }
 
-export const getConnectViemClient = ({
```

Comment:
Is it really a public client? I think it's more a hybrid, which is why I did not call it that way. I don't have a strong opinion on it thought